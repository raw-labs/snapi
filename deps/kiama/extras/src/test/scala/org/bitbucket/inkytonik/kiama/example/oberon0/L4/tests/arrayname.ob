MODULE arrayname;

TYPE
    atype1 = ARRAY 10 OF atype0;
    atype2 = ARRAY 3 * aconst OF INTEGER;

END arrayname.
