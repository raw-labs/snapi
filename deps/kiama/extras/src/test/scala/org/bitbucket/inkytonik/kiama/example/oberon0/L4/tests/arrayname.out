* _pp.ob
MODULE arrayname;

    TYPE
        atype1 = ARRAY 10 OF atype0;
        atype2 = ARRAY 3 * aconst OF INTEGER;

END arrayname.
* errors
src/test/scala/org/bitbucket/inkytonik/kiama/example/oberon0/L4/tests/arrayname.ob:4:26:error: atype0 is not declared
    atype1 = ARRAY 10 OF atype0;
                         ^
src/test/scala/org/bitbucket/inkytonik/kiama/example/oberon0/L4/tests/arrayname.ob:5:24:error: aconst is not declared
    atype2 = ARRAY 3 * aconst OF INTEGER;
                       ^
